Sources/Swift/SentryUserFeedbackConfiguration.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * The settings to use for how the user feedback form is presented, what data is required and how
+ * it's submitted, and some auxiliary hooks to customize the workflow.
+ */
+@available(iOSApplicationExtension 13.0, *)
+@objc public class SentryUserFeedbackConfiguration: NSObject {
+    /**
+     * Configuration settings specific to the managed widget that displays the UI form.
+     * - note: Default: `nil` to use the default widget settings.
+     */
+    public var widget: ((SentryUserFeedbackWidgetConfiguration) -> Void)?
```

Comment:
l: This name is a little misleading (as is uiForm). Starting the Sentry experience from the options would lead me to believe that this is the widget itself.

Suggestions:
- widgetConfigurator
- configureWidget

Or any name that makes it clear this will be an action rather than an object.